recipes/apache.rb

```diff
@@ -15,7 +15,7 @@
             password: node['kibana']['apache']['basic_auth_password'])
   owner node['apache']['user']
   group node['apache']['group']
-  mode 00644
+  mode 0o0644
```

Comment:
I don't know why, but tests not pass on ubuntu 12.04. look like something wrong with docker.